.gitignore

```diff
@@ -1,3 +1,7 @@
 /node_modules/
 /.eslintcache
-/build/
\ No newline at end of file
+/build/
+
+.DS_Store
```

Comment:
oh this rings a bell - you had me do something similar a while back - adding such things to my global user ignore file